db/scripts/dump.sql

```diff
@@ -0,0 +1,42 @@
+CREATE DATABASE IF NOT EXISTS AuctionDB;
+
+CREATE TABLE IF NOT EXISTS `AuctionDB`.`USERS` (
+`ID` BIGINT NOT NULL AUTO_INCREMENT,
+`NAME` VARCHAR(255) NULL,
+`PASSWORD` BINARY(60) NULL,
+PRIMARY KEY (`ID`),
+UNIQUE KEY (`NAME`),
+UNIQUE KEY (`PASSWORD`))
```

Comment:
If we will have unique value for password so in case user enters the value which exists he/she will know that such password already exists which can cause some security problems